v2.0/configure-replication-zones.md

```diff
@@ -398,6 +405,62 @@ $ cockroach start --insecure --host=<node6 hostname> --locality=datacenter=us-3
 
 There's no need to make zone configuration changes; by default, the cluster is configured to replicate data three times, and even without explicit constraints, the cluster will aim to diversify replicas across node localities.
 
+### Per-Replica Constraints to Specific Datacenters
+
+**Scenario:**
+
+- You have 5 nodes across 5 datacenters in 3 regions, 1 node in each datacenter.
+- You want data replicated 3 times, with a quorum of replicas for a database holding west coast data centered on the west coast and a database for nationwide data replicated across the entire country.
+
+**Approach:**
+
+1. Start each node with its region and datacenter location specified in the `--locality` flag:
+
+    ~~~ shell
+    # Start the four nodes:
+    $ cockroach start --insecure --host=<node1 hostname> --locality=region=us-west1,datacenter=us-west1-a
+    $ cockroach start --insecure --host=<node2 hostname> --locality=region=us-west1,datacenter=us-west1-b \
+    --join=<node1 hostname>:27257
+    $ cockroach start --insecure --host=<node3 hostname> --locality=region=us-central1,datacenter=us-central1-a \
+    --join=<node1 hostname>:27257
+    $ cockroach start --insecure --host=<node4 hostname> --locality=region=us-east1,datacenter=us-east1-a \
+    --join=<node1 hostname>:27257
+    $ cockroach start --insecure --host=<node4 hostname> --locality=region=us-east1,datacenter=us-east1-b \
+    --join=<node1 hostname>:27257
+    ~~~
+
+2. On any node, configure a replication zone for the database used by the west coast application:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Create a YAML file with the replica count set to 5:
+    $ cat west_app_zone.yaml
+    ~~~
+
+    ~~~
+    constraints: {"+region=us-west1": 2, "+region=us-central1": 1}
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    # Apply the replication zone to the database used by the west coast application:
+    $ cockroach zone set west_app_db --insecure -f west_app_zone.yaml
+    ~~~
+
+    ~~~
+    range_min_bytes: 1048576
+    range_max_bytes: 67108864
+    gc:
+      ttlseconds: 86400
+    num_replicas: 3
+    constraints: {+region=us-west1: 2, +region=us-central1: 1}
+    constraints: {+region=us-central1: 1, +region=us-west1: 2}
```

Comment:
This looks like the same constraint but reversed. Is this needed?

Reply:
Whoops, the first `constraints` line should be deleted.

Reply:
Done.